l10n_eu_adr_report/static/src/scss/adr_report.scss

```diff
@@ -0,0 +1,28 @@
+
+.adr-report table,
+.adr-report td {
+    border: 1px solid grey;
+    padding: 0.25rem;
+  }
+.adr-report table{
+    margin-bottom: 0px;
+}
+.adr-report{
+  font-size: 12px;
+  }
+.adr-text{
+  white-space: pre-line;
+}
+.wide-col{
+  width: 40%;
+}
+.uppercase{
+  text-transform: uppercase;
+}
+.emphatic{
+  font-weight: bold;
+}
+h5{
+  text-align: right;
```

Comment:
:warning: this will be applied to all reports (if you don't wrap them all as mentioned above)